.drone.star

```diff
@@ -669,3 +671,43 @@ def githubComment(alternateSuiteName):
             ],
         },
     }]
+
+def skipIfUnchanged(ctx, type):
+    if ("full-ci" in ctx.build.title.lower()):
+        return []
+
+    skip_step = {
+        "name": "skip-if-unchanged",
+        "image": OC_CI_DRONE_SKIP_PIPELINE,
+        "when": {
+            "event": [
+                "pull_request",
+            ],
+        },
+    }
+
+    base_skip_steps = [
+        "^.github/.*",
+        "^changelog/.*",
+        "README.md",
+    ]
+
+    if type == "gui-tests":
+        gui_files = [
+            "^src/.*",
+        ]
```

Comment:
I think gui test files are all inside `test` dir. Can you run gui tests when there is any change in either `test` or `src` dir. And then run unit tests when there is change in `src` dir?